models/product-data/src/transformers.ts

```diff
@@ -0,0 +1,102 @@
+import {
+  CategoryReference,
+  ProductVariant,
+  SearchKeyword,
+} from '@commercetools/platform-sdk';
+import * as Category from '@commercetools-test-data/category';
+import { TCategoryGraphql } from '@commercetools-test-data/category';
+import { LocalizedString, Reference } from '@commercetools-test-data/commons';
+import { Transformer } from '@commercetools-test-data/core';
+import { faker } from '@faker-js/faker';
+import type { TProductData, TProductDataGraphql } from './types';
+
+const transformers = {
+  default: Transformer<TProductData, TProductData>('default', {
+    buildFields: [
+      'name',
+      'categories',
```

Comment:
Should `categoryOrderHints`, `masterVariant` and `searchKeywords` be included here?

Reply:
`categoryOrderHints` doesn't have a model, but I'll definitely include `masterVariant` and `searchKeywords`!

See ed791e8ff8fa66a4ec517db7b23bd5fd94fb63ea